### DISCUSSION TITLE
Manual deletion of a specific day

### DISCUSSION THREAD

**Discussion by a-elkaim** [blakeblackshear/frigate#14400] [General Support]
Hello everyone,

I was using Synology Surveillance Station until recently, sick of the false positive, I bought a Google Coral and install frigate, and it has simply been amazing so far.

That being said, I had a specific workflow using Surveillance Station that I cannot replicate using Frigate, or at least I haven't figured out how to do so.

With surveillance station, I kept recordings ''forever'', and every once in a while, I went through the footage and saved footage of my kids or my pets doing funny stuff, and I make some kind of complication to show our relatives.

After reviewing events from a certain period and exporting those I wanted to save, I deleted the recordings from said period, saving quite a lot of space in the process.

Now, using Frigate, it doesn't seem possible to manually delete a specific day's recordings.

I can obviously delete the recording from the hard drive, but this results in a database corruption afterwards.

So, using Frigate is it possible to only delete recordings and events from a specific time period ?

I thank you in advance for your help!

**Comment by NickM-27** [comment]
You can do this by selecting all review items in a day then pressing delete in the UI.

**Reply by a-elkaim** [reply]
Will this delete the original recording as well or only the events entries ?

Also, is there a specific configuration required for recording retention in order to achieve this behavior ?

**Reply by NickM-27** [reply]
Yes, it deletes the recordings like it says in the dialog

**Reply by a-elkaim** [reply]
Perfect, I will try this out.

Also, what happens to recording where no motion is detected, does it get deleted as well ?

**Reply by NickM-27** [reply]
No, if you have continuous recordings that don't overlap with the review items those will not be deleted

**Reply by a-elkaim** [reply]
What would be the best way of deleting those then (given my specific use case) ?

**Reply by NickM-27** [reply]
if you truly want to do that, then just delete from the storage manually, the database will not be corrupted
